var/spack/repos/spack_repo/builtin/packages/hpx/package.py

```diff
@@ -211,8 +211,6 @@ class Hpx(CMakePackage, CudaPackage, ROCmPackage):
     conflicts("~generic_coroutines", when="target=aarch64:", msg=_msg_generic_coroutines_target)
     conflicts("~generic_coroutines", when="target=arm:", msg=_msg_generic_coroutines_target)
 
-    # Patches APEX
-    patch("git_external.patch", when="@1.3.0 instrumentation=apex")
```

Comment:
the related patch can now be deleted as well.

Reply:
Removed